contents/docs/experiments/exposures.mdx

```diff
@@ -0,0 +1,134 @@
+---
+title: Exposures
+sidebar: Docs
+showTitle: true
+---
+
+import { CalloutBox } from 'components/Docs/CalloutBox'
+import { ProductScreenshot } from 'components/ProductScreenshot'
+
+Exposures are the foundation of experiment analysis in PostHog. A user must be **exposed** to your experiment before they can be included in any metric calculations. Understanding how exposures work is crucial for running successful experiments.
+
+## What is an exposure?
+
+An exposure occurs when a user encounters the part of your product where the experiment is running. This is the moment they become a participant in your experiment and start contributing to your metrics.
+
+**For most users**: If you're using PostHog feature flags with our SDKs, exposures are tracked automatically. When you call `getFeatureFlag()`, PostHog sends a `$feature_flag_called` event with all the necessary properties. You don't need to do anything extra.
+
+**Technical details**: PostHog considers a user exposed when it receives a `$feature_flag_called` event containing:
+- The property `$feature_flag` matching your experiment's flag key
+- The property `$feature_flag_response` with a variant value (e.g., "control" or "test")
+
+Only exposed users are included in your experiment analysis. Any events that occur before exposure are ignored, ensuring clean and accurate results.
+
+<CalloutBox icon="IconInfo" title="Important: Exposure comes first" type="info">
```

Comment:
```suggestion
<CalloutBox icon="IconInfo" title="Exposure comes first" type="info">
```

Important is redundant in a callout. You cal always upgrade this to a warning or sm other type of more grave callout